src/sdk/java/pom.xml

```diff
@@ -3,15 +3,40 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com._4paradigm.sql</groupId>
-    <artifactId>sql-sdk</artifactId>
+    <groupId>com.4paradigm.hybridsql</groupId>
+    <artifactId>fedb-sdk</artifactId>
     <version>2.2.0</version>
     <modules>
-           <module>sql-jdbc</module>
-           <module>sql-native</module>
+           <module>fedb-jdbc</module>
+           <module>fedb-native</module>
     </modules>
     <packaging>pom</packaging>
-    <name>sdk</name>
+    <name>fedb-sdk</name>
+    <description>A NewSQL database optimised for online inferencing and decision making applications.</description>
+    <url>https://github.com/4paradigm/fedb</url>
+     <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+    <developers>
+        <developer>
+            <id>jingchen2222</id>
```

Comment:
Should you change the developer name? @dl239

Reply:
yes